queries/water-z16.pgsql

```diff
@@ -34,6 +34,24 @@ FROM
         mz_is_water = TRUE
         AND way && !bbox!
 
+    --
+    -- Water line geometries
+    --
+    UNION
+
+    SELECT name,
+           NULL AS area,
+           COALESCE("waterway", "natural", "landuse") AS kind,
```

Comment:
If `waterway` is guaranteed to not be null based on the `WHERE` clause, then there's no need for this `COALESCE`, since it will never get past the first column, right?